### DISCUSSION TITLE
Running NekRS on intel Arc GPGPUs

### DISCUSSION THREAD

**Discussion by dcoxe93** [Nek5000/nekRS#481] [General]
Hi all,

Is there any benchmark or specific process for running nekrs on an intel arc a770/750 gpu?  Looking at their double precision performance/price they seem like a decent option I was just curious if there is a way to make sure nekRS can utilize these GPUs?

Thanks,
Daniel

**Comment by stgeke** [comment]
The upcoming release (v23.0) will add DPC++ support enabling you to run on Intel GPUs. However I am not sure if this consumer GPU will be able to perform well for our kind of workloads without tuning. It doesn't seem to support FP64 in hardware. That's an issue because we currently support running nekRS in FP32.

**Reply by dcoxe93** [reply]
Good to know! What is the timeline for v23?
